src/server/http/http.go

```diff
@@ -45,6 +46,15 @@ func New(port uint16, pachClientFactory func(ctx context.Context) *client.APICli
 	// JSON schemas.
 	mux.Handle("/jsonschema/", http.StripPrefix("/jsonschema/", http.FileServer(http.FS(jsonschema.FS))))
 
+	// GRPC gateway.
+	gwmux := restgateway.NewRestGatewayMux(pachClientFactory)
+	if gwmux == nil {
+		log.Error(context.Background(), "httpserver: failed to register rest api handlers with grpc-gateway")
+		return nil
+	}
+	mux.Handle("/api/", http.StripPrefix("/api", gwmux))
+	log.Info(context.Background(), "httpserver: completed grpc gateway rest api registrations")
```

Comment:
Use pctx.Background() or pctx.TODO() in preference to context.Background() or context.TODO().

In this case, since NewMux does IO, it will need a real context passed into http.New.  There is one available at the places where this is called, so no big deal to do that refactor.